src/chainstate/stacks/boot/pox-2.clar

```diff
@@ -831,7 +887,16 @@
                     add-amount: increase-by })))
             (err ERR_STACKING_UNREACHABLE))
       ;; NOTE: stacking-state map is unchanged: it no longer tracks amount-stacked in PoX-2
-      (ok { stacker: tx-sender, total-locked: (+ amount-stacked increase-by)})))
+
+      ;; emit event
+      (print (merge event-info {
+            name: "stack-increase", 
+            args: {
+                increase-by: increase-by, 
```

Comment:
do these args need to be accounted for the test?

Reply:
Yes -- I think that the unit test coverage for these new events should verify that the data returned is what we expect.  There's not going to be a way to change this code once 2.1 is live.